Client/Frontend/Browser/Browser.swift

```diff
@@ -117,6 +117,8 @@ class Browser: NSObject, BrowserWebViewDelegate {
             configuration!.userContentController = WKUserContentController()
             configuration!.preferences = WKPreferences()
             configuration!.preferences.javaScriptCanOpenWindowsAutomatically = false
+            configuration!.allowsInlineMediaPlayback = true
+            configuration!.mediaPlaybackAllowsAirPlay = true
```

Comment:
yeah, I didn't mention it in the bug; I should have. oh, good call about the iOS 9 deprecation.